distribution/changelog.txt

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Fix: [#12691] Ride graph tooltip incorrectly used count instead of number string.
 - Fix: [#12694] Crash when switching ride types with construction window open.
+- Fix: [#12707] Incomplete loop collision box allowed overlapping track (original bug).
```

Comment:
```suggestion
- Fix: Incomplete loop collision box allowed overlapping track (original bug).
```

The number we quote in here is of an issue in the GitHub tracking that got fixed. Since this was not tracked (apparently, I did not search), then no need for the number.

Also, can you reword your commit to be what is here in the changelog? I think it's a lot nicer and more informative:
"Fix: Incomplete loop collision box allowed overlap (original bug)"